src/lib/components/viewer/AnnotationLayer.svelte

```diff
@@ -200,34 +195,40 @@ Assumes it's a child of a ViewerContext
       href={getViewerHref({ document, note, mode: $mode, embed })}
       class="note-highlight {note.access}"
       class:active={$currentNote?.id === note.id}
-      title={note.title}
       style:top="{note.y1 * 100}%"
       style:left="{note.x1 * 100}%"
       style:width="{width(note) * 100}%"
       style:height="{height(note) * 100}%"
+      title={note.title}
       on:click={(e) => openNote(e, note)}
     >
       {note.title}
     </a>
   {/each}
 
-  {#if $currentNote && !Boolean($newNote) && $currentNote.page_number === page_number}
+  {#if $currentNote && !Boolean($newNote) && $currentNote.page_number === page_number && !isPageLevel($currentNote)}
     <div
       class="note card"
-      style={positionNote($currentNote, 1.5)}
+      style={positionNote($currentNote, 0.5)}
       transition:fly={{ duration: 250, y: "-1.1rem" }}
     >
-      {#if writing}
-        <EditNote
-          note={$currentNote}
-          {document}
-          {page_number}
-          on:close={closeNote}
-          on:success={(e) => onEditNoteSuccess(e, $currentNote)}
-        />
-      {:else}
-        <Note note={$currentNote} {scale} on:close={closeNote} />
-      {/if}
+      <Note note={$currentNote} {scale} on:close={closeNote} />
+    </div>
+  {/if}
+
+  {#if writing && $currentNote}
+    <div
+      class="note card"
+      style={positionNote($currentNote, 0.5)}
+      transition:fly={{ duration: 250, y: "-1.1rem" }}
+    >
+      <EditNote
+        note={$currentNote}
+        {document}
+        {page_number}
+        on:close={closeNote}
+        on:success={(e) => onEditNoteSuccess(e, $currentNote)}
+      />
```

Comment:
Now that these are decoupled, editing a note will both open the read-version and the write-version simultaneously.

<img width="781" alt="Screenshot 2025-03-20 at 15 37 19" src="https://github.com/user-attachments/assets/3a881175-e014-4764-97fb-a4f771c10612" />